st2actions/st2actions/runners/windows_script_runner.py

```diff
@@ -151,6 +162,37 @@ def _run_script(self, script_path):
 
         return exit_code, stdout, stderr, timed_out
 
+    def _get_script_arguments(self, named_args=None, positional_args=None):
+        """
+        Builds a string of named and positional arguments in PowerShell format,
+        which are passed to the script.
+
+        :param named_args: Dictionary with named arguments
+        :type named_args: ``dict``.
+
+        :param positional_args: List of positional arguments
+        :type positional_args: ``str``
+
+        :rtype: ``str``
+        """
+        cmd_parts = []
+        if positional_args:
+            cmd_parts.append(positional_args)
+        if named_args:
+            for (arg, value) in six.iteritems(named_args):
+                arg = quote_windows(arg)
+                if value is None or (isinstance(value, six.string_types) and len(value) < 1):
+                    LOG.debug('Ignoring arg %s as its value is %s.', arg, value)
+                    continue
+                if isinstance(value, bool):
+                    if value:
+                        cmd_parts.append("-%s" % (arg))
```

Comment:
As far as "titlelizing" parameter names go - it sounds like this could be a boolean windows script runner parameter (e.g. "titlelize_named_parameters" or something better :P).